src/global/CMakeLists.txt

```diff
@@ -10,4 +10,7 @@ target_link_libraries(global-static common)
 
 add_library(global STATIC
   $<TARGET_OBJECTS:libglobal_objs>)
-target_link_libraries(global ceph-common ${EXTRALIBS})
+target_link_libraries(global
```

Comment:
why does `global` need `erasure_code`?

Reply:
https://github.com/ceph/ceph/blob/master/src/global/global_init.cc#L23
https://github.com/ceph/ceph/blob/master/src/global/global_init.cc#L565

A previous version avoided linking ``ceph-common`` against ``erasure-code``, so it was needed here. No longer the case though.